arelle/plugin/validate/ESEF/Util.py

```diff
@@ -93,4 +95,21 @@ def checkImageContents(modelXbrl, imgElt, imgType, isFile, data):
                 _("Image type %(imgType)s has wrong header type: %(headerType)s"),
                 modelObject=imgElt, imgType=imgType, headerType=headerType,
                 messageCodes=("ESEF.2.5.1.imageDoesNotMatchItsFileExtension", "ESEF.2.5.1.incorrectMIMETypeSpecified"))
+            
+def resourcesFilePath(modelManager, fileName):
+    # resourcesDir can be in cache dir (production) or in validate/EFM/resources (for development)
```

Comment:
This was copied over from the EFM plugin. Im wondering if there is a way that we could pull this function out into a `validate/utils.py` file so that it can be shared across multiple validation plugins rather than copy thing code where ever we need it?